docs/acme-divergences.md

```diff
@@ -4,9 +4,14 @@ While Boulder attempts to implement the ACME specification as strictly as possib
 
 Since Boulder evolved alongside the ACME specification there is not one exact ACME draft number that can be referenced in isolation to understand the protocol Boulder speaks. This document exists to detail differences between what Boulder does and the most-recently published ACME draft. Since ACME is not yet finalized it will be updated as new numbered drafts are published.
 
-We refer to what Boulder presently implements as "ACME v1" and we are [actively working on "ACME v2"](https://letsencrypt.org/2017/06/14/acme-v2-api.html), a second API endpoint with less divergences that is based on what will become the final ACME RFC.
+**ACME v2 divergences from [`draft-ietf-acme-acme-10`](https://tools.ietf.org/html/draft-ietf-acme-acme-10).**
 
-**Current draft: [`draft-ietf-acme-acme-07`](https://tools.ietf.org/html/draft-ietf-acme-acme-07).**
+Presently the following protocol features are not implemented:
+
+- Pre-authorization. This is an optional feature and we have no plans to implement it. V2 clients should use order based issuance without pre-authorization.
+- The “orders” field on account objects. We intend to support this non-essential feature in the near future. Please follow Boulder Issue [#3335](https://github.com/letsencrypt/boulder/issues/3335).
```

Comment:
Can you replace the smartquotes around "orders" : `“orders”`

Reply:
I think it's better now